### PR TITLE
feat(budget): apply design qa

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA
- quest: https://pyengine.atlassian.net/browse/QA-766

### CODE REVIEW

**Review by yuda110 — APPROVED** [review]
LGTM!!

File: apps/web/src/services/cost-explorer/budget/budget-detail/modules/budget-info/BudgetDetailInfo.vue [hunk 1]
```diff
@@ -87,26 +124,51 @@ const getTargetLabel = (projects: ProjectReferenceMap) => {
...
+                >
+                    <span class="cost-type-content">{{ state.processedProviderValue }}</span>
+                </span>
+                <p-text-button v-if="state.isTextTruncate"
+                               style-type="highlight"
+                               @click="handleClickViewAll"
+                >
+                    {{ $t('BILLING.COST_MANAGEMENT.BUDGET.VIEW_ALL') }}
+                </p-text-button>
+            </div>
```
Comment (sulmoJ) [inline apps/web/src/services/cost-explorer/budget/budget-detail/modules/budget-info/BudgetDetailInfo.vue:171]: Issue: Oh... I'm so sorry. I didn't know it was a design feature and I removed it.
Additionally, it seems to be displayed as a popover like this.
image

**Review by sulmoJ — APPROVED** [review]
I left a Issue.
LGTM!!

**Review by WANZARGEN — APPROVED** [review]
Actually It's really hard to compare with the requirement.. 🥲
Please split into the several commits 🙏.
I've just checked the convention.

**Pull request merged**